### PR TITLE
opa configuration in spinnaker

### PR DESCRIPTION
**Pull request by yeshaswiniOpsmx**

### CODE REVIEW

File: charts/oes/values.yaml [hunk 1]
```diff
@@ -572,7 +582,7 @@ spinnaker:
       fiat.yml:
         artifactId: docker.io/opsmxdev/ubi8-oes-fiat:1.11.0
       front50.yml:
-        artifactId: docker.io/opsmxdev/ubi8-oes-front50:0.24.0
+        artifactId: docker.io/opsmxdev/ubi8-oes-front50:0.25.1-opa
```
Comment (abhinayopsmx) [inline charts/oes/values.yaml:585]: Can you move all these spinnaker images to Quay and change these artifactIds so that images will be fetched from Quay?

**Review by abhinayopsmx — COMMENTED** [review]
Fix merge conflicts, let me know if you need any help.

File: charts/oes/values.yaml (hunk 1 above)
Comment (yeshaswiniOpsmx) [reply charts/oes/values.yaml]: Done

**Review by abhinayopsmx — APPROVED** [review]
LGTM, please go ahead and merge if already tested

**Pull request merged**